suitcase/tests/data/temperature_data.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import division
+import uuid
+
+import numpy as np
+from functools import wraps
+
+# "Magic numbers" for this simulation
+start, stop, step, points_per_step = 0, 6, 1, 7
+deadband_size = 0.9
+num_exposures = 17
+
+
+def example(func):
+    @wraps(func)
+    def mock_run_start(mds, run_start_uid=None, sleep=0, make_run_stop=True):
```

Comment:
As you might remember, all of this stuff -- the whole temperature ramp example -- was written before bluesky even existed. Back then, the only way to compose Documents to test metadatastore/databroker against was to assemble them manually like this. Now, we can do it much more succinct (and correctly) using bluesky. This old code of mine was deleted in favor of that approach, and I'd prefer that it stay gone.

Reply:
good point. it is better not to rewrite example data each time.